src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -408,8 +408,7 @@ def set_feedback(feedback):
 def set_cli_recommendation(api_version, feedback):
     # This function returns the user's selection and feedback on the cli-recommendation results
     # Please refer to feedback_design.md of cli-recommendation for detailed information
-    _session.cli_recommendation = {"api_version": api_version,
-                                   "feedback": feedback}
+    _session.cli_recommendation = json.dumps({"api_version": api_version, "feedback": feedback})
```

Comment:
Please add a comment to explain why we need `json.dumps` here

Reply:
already modified, ready for rereview.

Reply:
already modified in d0067d0c3aee83e97de677f5f377bcdbcccb7927